modules/xfem/test/tests/heat_flux_bc/tests

```diff
@@ -0,0 +1,13 @@
+[Tests]
+  design = 'XFEMConvectiveHeatFluxBC.md'
+  issues = '#17331'
+  [2d]
+    type = Exodiff
+    input = 2d.i
+    exodiff = '2d_out.e'
+    map = false
+    unique_id = true
+    requirement = 'The XFEM module shall provide the capability to model convective heat flux across '
```

Comment:
What do you mean by "cross". It sounds like there is a flux going from one side to the other side, but you actually apply a heat flux on either side of the interface independently.

Reply:
I reworded it to avoid confusion.